JacksonDunstanNativeCollections/NativeElement.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Unity.Collections;
+using Unity.Collections.LowLevel.Unsafe;
+
+/// <summary>
+/// NativeElement is a wrapper for a single blittable element to be passed between managed and unmanaged code in jobs
+/// </summary>
+/// <typeparam name="T">The blittable type to be stored</typeparam>
+[StructLayout(LayoutKind.Sequential)]
+[NativeContainer]
+[DebuggerDisplay("{Element}")]
+public unsafe struct NativeElement<T> : IDisposable where T : unmanaged
+{
+    // The pointer to the element
+    [NativeDisableUnsafePtrRestriction]
+    T* m_ptr;
+
+#if ENABLE_UNITY_COLLECTIONS_CHECKS
+    // Safety
+    AtomicSafetyHandle m_Safety;
+
+    [NativeSetClassTypeToNullOnSchedule]
+    DisposeSentinel m_DisposeSentinel;
+#endif
+
+    // The allocator label
+    Allocator m_AllocatorLabel;
+
+    // Constructor which only takes a label and then calls the main constructor with an instance of T
+    public NativeElement(Allocator label) : this(new T(), label) { }
+
+    // Main constructor logic that takes an element and a label
+    public NativeElement(T element, Allocator label)
```

Comment:
Rename `element` to `initialValue` for consistency with other types, move it to the end of the parameter list, and add a default value: `T initialValue = default(T)`.

Reply:
Rename `label` to `allocator` for consistency with other types.